<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>input_image</class>
 <widget class="QWidget" name="input_image">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="2" column="0">
    <widget class="QCheckBox" name="enable_checkbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Enable </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="import_button">
     <property name="text">
      <string>Import</string>
     </property>
     <property name="icon">
      <iconset theme="folder"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGraphicsView" name="image_container"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="delete_button">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset theme="user-trash"/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="use_grid_image_as_input_checkbox">
     <property name="text">
      <string>Use grid image as input</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>import_button</sender>
   <signal>clicked()</signal>
   <receiver>input_image</receiver>
   <slot>import_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delete_button</sender>
   <signal>clicked()</signal>
   <receiver>input_image</receiver>
   <slot>delete_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enable_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>enabled_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_grid_image_as_input_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>input_image</receiver>
   <slot>use_grid_image_as_input_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>import_clicked()</slot>
  <slot>delete_clicked()</slot>
  <slot>enabled_toggled(bool)</slot>
  <slot>use_grid_image_as_input_toggled(bool)</slot>
 </slots>
</ui>
